aten/src/ATen/native/native_functions.yaml

```diff
@@ -6000,14 +6000,20 @@
     CPU: legacy::cpu::_th_geqrf
     CUDA: legacy::cuda::_th_geqrf
 
-- func: orgqr.out(Tensor self, Tensor input2, *, Tensor(a!) out) -> Tensor(a!)
+- func: householder_product(Tensor input, Tensor tau) -> Tensor
+  variants: function
+  dispatch:
+    CPU: householder_product
+
+- func: householder_product.out(Tensor input, Tensor tau, *, Tensor(a!) out) -> Tensor(a!)
   dispatch:
-    CPU: orgqr_out
+    CPU: householder_product_out
 
+# orgqr, alias for householder_product
 - func: orgqr(Tensor self, Tensor input2) -> Tensor
   variants: method, function
-  dispatch:
-    CPU: orgqr
+
+- func: orgqr.out(Tensor self, Tensor input2, *, Tensor(a!) out) -> Tensor(a!)
```

Comment:
So orgqr was always CPU-only and this is also CPU-only for now?

Reply:
Yes, it was always CPU-only. CUDA support is in https://github.com/pytorch/pytorch/pull/51348 which now passes all CI checks and can be merged. After https://github.com/pytorch/pytorch/pull/51348 is merged I'll update this PR.